tests/golden/builtin/builtin/builtin/01_keycloak_helmchart/postgresql/templates/primary/statefulset.yaml

```diff
@@ -111,7 +109,7 @@ spec:
           periodSeconds: 10
           successThreshold: 1
           timeoutSeconds: 5
-        name: keycloak-postgresql
+        name: postgresql
```

Comment:
I feel that "keycloak-postgresql" isn't a wrong name, as it is the statefulset for postgres for keycloak.

Reply:
I don't know, because this is inherited by the helm chart change and therefore we have no influence on that.